addons/website/static/src/snippets/s_dynamic_snippet_carousel/000.xml

```diff
@@ -15,7 +15,7 @@
             <!-- Content -->
             <t t-set="rowSize" t-value="chunkSize" />
             <t t-set="slideSize" t-value="rowSize * rowPerSlide" />
-            <t t-set="colClass" t-value="'d-flex flex-grow-0 flex-shrink-0 col-12 col-lg-' + Math.trunc(12 / rowSize).toString()"/>
+            <t t-set="colClass" t-value="'d-flex flex-grow-0 flex-shrink-0 col-12 col-sm-' + Math.trunc(12 / rowSize).toString()"/>
```

Comment:
Is this really necessary ? 
It seems when the viewport reaches the small size, the carousel is rendered again with a single element per row and the column classes are also adapted.
![Capture d’écran 2024-10-01 à 15 58 40](https://github.com/user-attachments/assets/f82da9ac-d776-4887-b690-15ddb07a6936)

I would go back on what as been done before (https://github.com/odoo/odoo/pull/135509/files#diff-2334540c8bf2687bd922610b4399122c7d87ef9f60f1e90560ffabdd6ac8a3b8L18)
```suggestion
            <t t-set="colClass" t-value="'d-flex flex-grow-0 flex-shrink-0 col-' + Math.trunc(12 / rowSize).toString()"/>
```

Reply:
indeed thanks 👍